console/src/main/java/org/georchestra/console/ws/backoffice/users/UserInfoExporterImpl.java

```diff
@@ -0,0 +1,273 @@
+        csv.append(toFormatedString(account.getDescription()));
+        csv.append(CSV_DELIMITER);// Job Title
+        csv.append(CSV_DELIMITER);// Department
+        csv.append(CSV_DELIMITER);// Office Location
+        csv.append(CSV_DELIMITER);// Organizational ID Number
+        csv.append(CSV_DELIMITER);// Profession
+        csv.append(CSV_DELIMITER); // Account
+        csv.append(toFormatedString(account.getPostalAddress()));
+        csv.append(CSV_DELIMITER);// Business Address
+        csv.append(toFormatedString(account.getStreet()));
+        csv.append(CSV_DELIMITER);// Business Street
+        csv.append(CSV_DELIMITER);// Business Street 2
+        csv.append(CSV_DELIMITER); // Business Street 3
+        csv.append(toFormatedString(account.getPostOfficeBox()));
+        csv.append(CSV_DELIMITER);// Business Address PO Box
+        csv.append(CSV_DELIMITER);// Business City
+        csv.append(CSV_DELIMITER);// Business State
+        csv.append(toFormatedString(account.getPostalCode()));
+        csv.append(CSV_DELIMITER); // Business Postal Code
+        csv.append(toFormatedString(account.getStateOrProvince()));
+        csv.append(CSV_DELIMITER);// Business Country
+        csv.append(CSV_DELIMITER);// Other Phone
+        csv.append(CSV_DELIMITER);// Other Fax
+        csv.append(toFormatedString(account.getRegisteredAddress()));
+        csv.append(CSV_DELIMITER); // Other Address
+        csv.append(toFormatedString(account.getPhysicalDeliveryOfficeName()));
+        csv.append(CSV_DELIMITER);// Other Street
+        csv.append(CSV_DELIMITER);// Other Street 2
+        csv.append(CSV_DELIMITER);// Other Street 3
+        csv.append(CSV_DELIMITER);// Other Address PO Box
+        csv.append(CSV_DELIMITER); // Other City
+        csv.append(CSV_DELIMITER);// Other State
+        csv.append(CSV_DELIMITER);// Other Postal Code
+        csv.append(CSV_DELIMITER);// Other Country
+        csv.append(CSV_DELIMITER); // Callback
+        csv.append(CSV_DELIMITER);// Car Phone
+        csv.append(CSV_DELIMITER);// ISDN
+        csv.append(CSV_DELIMITER);// Radio Phone
+        csv.append(CSV_DELIMITER);// TTY/TDD Phone
+        csv.append(CSV_DELIMITER); // Telex
+        csv.append(CSV_DELIMITER);// User 1
+        csv.append(CSV_DELIMITER);// User 2
+        csv.append(CSV_DELIMITER);// User 3
+        csv.append(CSV_DELIMITER); // User 4
+        csv.append(CSV_DELIMITER);// Keywords
+        csv.append(CSV_DELIMITER);// Mileage
+        csv.append(CSV_DELIMITER);// Hobby
+        csv.append(CSV_DELIMITER);// Billing Information
+        csv.append(CSV_DELIMITER); // Directory Server
+        csv.append(CSV_DELIMITER);// Sensitivity
+        csv.append(CSV_DELIMITER);// Priority
+        csv.append(CSV_DELIMITER);// Private
+        csv.append(CSV_DELIMITER); // Categories
+        csv.append("\r\n"); // CRLF
+        return csv.toString();
+    }
+
+    private String toFormatedString(String data) {
```

Comment:
I am wondering whether a dto should be afforded to contains null or just empty string (faced the same kind of indirection when serializing to json, also wondering what is preferred at gui side, missing key in json, empty string or null).

Reply:
Finished the migration from Account to UserInfoExporter as per 35fb120dba3a710be0e4500c673bea8a8df94b86. Resolving this conversation as it's outside the scope of this PR (unrelated to GDPR and existing, old code)